src/applications/claims-status/components/AddFilesForm.jsx

```diff
@@ -163,14 +178,13 @@ class AddFilesForm extends React.Component {
 
   render() {
     return (
-      <div>
-        <div>
-          <p>
```

Comment:
This `p` was causing issues with DOM validation. Apparently `p` and `ol` tags cannot be nested inside of `p` tags, both of which are part of the `va-additional-info` DOM